src/macros/image-card.njk

```diff
@@ -0,0 +1,6 @@
+{% macro image_card(name)%}
```

Comment:
@will0101 Nice! I have never used macros. This is great. What do you think about a more specific name though? We'll also be adding some type of title for each person, so this macro is going to be specific to people. What about `person-card` for the file and macro name? Also note the hyphen instead of the underscore to stay consistent kebab case.

Reply:
I changed the name of the macro also the file. But, using a hyphen in the name of the macro throws an error.

Reply:
Ah, bummer. I guess since it's Python port. What's the error?

Reply:
`expected arguments
`Template render error` was thrown:`